core/src/raw/http_util/client.rs

```diff
@@ -167,11 +168,13 @@ impl HttpClient {
         })?;
 
         // Get content length from header so that we can check it.
-        // If the request method is HEAD, we will ignore this.
-        let content_length = if is_head {
+        //
+        // - If the request method is HEAD, we will ignore content length.
+        // - If response contains content_encoding, we should omit it's content length.
```

Comment:
I propose that update the comment to mention that this trick would only be available in WASM32

Reply:
Welcome to submit a PR to improve it :love_letter: